### DISCUSSION TITLE
Contribution to degradation by submodels

### DISCUSSION THREAD

**Discussion by kethan27** [pybamm-team/PyBaMM#3139] [Q&A]
Can we find contribution of each sub model like  "SEI": "reaction limited",
         "SEI porosity change": "true",
        "lithium plating": "partially reversible",
         "lithium plating porosity change": "true", 
         "particle mechanics": ("swelling and cracking"),
        "SEI on cracks": "true",
        "loss of active material": "reaction-driven",
        "SEI film resistance": "distributed",
        "SEI": "reaction limited",
        "SEI film resistance": "distributed",
        "SEI on cracks":"true",
        "SEI porosity change":"true",  in capacity fade?. If yes how to do it

**Comment by brosaplanella** [comment]
You mean the effect of each option into the capacity, you can just compare various models (similarly to how it is done in [this notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/getting_started/tutorial-2-compare-models.html)) and plot the summary variables (similar to [this notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/simulating-long-experiments.html))

**Reply by alihassan90** [reply]
docs.pybamm.org is not working. Didnt want to start a new discussion so just mentioning here. I hope it gets back online.
